### PR TITLE
feat: Added custom keep-ilert integration API support to Keep

### LINKED ISSUE #1496: [🔨 Enhancement]: Ilert provider to support official Keep integration
Ilert now supports Keep officially from their platform 🥳 
CleanShot 2024-08-06 at 15 25 11@2x

Raw details:
The URL is POST [https://api.ilert.com/api/v1/events/keep/{ALERT-SOURCE-API-KEY}](https://api.ilert.com/api/v1/events/keep/%7BALERT-SOURCE-API-KEY%7D)
The request body is equal to our standard event API https://api.ilert.com/api-docs/#tag/events/post/events (you should not send the apiKey body field).
Additional custom fields should be piped or added in the customDetails: {} field.

DoD for this issue:
1. Adjust `_notify` to support the ilert custom Keep integration
2. Adjust documentation to explain how to add Keep to ilert.

Any further instructions/help is always available in our [Slack](https://slack.keephq.dev)

### PR DESCRIPTION
**Pull request by Mubashirshariq**
/claim #1496 

Closes #1496

## 📑 Description
added custom keep integration api to keep.

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: keep/providers/ilert_provider/ilert_provider.py [hunk 1]
```diff
@@ -307,12 +307,11 @@ def __post_ilert_event(
             "images": images,
             "links": links,
             "customDetails": custom_details,
-            "routingKey": routing_key,
         }
         self.logger.info("Posting Ilert event", extra=payload)
-        payload["apiKey"] = self.authentication_config.ilert_token
+        # payload["apiKey"] = self.authentication_config.ilert_token
```
Comment (talboren) [inline keep/providers/ilert_provider/ilert_provider.py:312]: remove redundant comments

Comment (talboren) [inline keep/providers/ilert_provider/ilert_provider.py:312]:
```suggestion
```

**Review by talboren — APPROVED** [review]
LGTM

**Review by VladimirFilonov — APPROVED** [review]
LGTM

**Pull request merged**